pkg/profiler/cpu/cpu.go

```diff
@@ -158,6 +161,9 @@ func NewCPUProfiler(
 
 		memlockRlimit: memlockRlimit,
 
+		// increase cache length if needed to track more errors
+		processErrorTracker: cache.NewLRUCache[string, int](prometheus.WrapRegistererWith(prometheus.Labels{"cache": "error_tracker"}, reg), 1),
```

Comment:
1 entry is not going to be enough, since this can happen for X amount of processes, and even N amount of mappings within those processes

Reply:
Also let's choose a better name `error_tracker` is a bit too generic, maybe `no_text_section_error_tracker`?

Reply:
Cool I was thinking maybe we could eventually reuse the cache to log different errors too but no strong opinions :D